packages/mongodb-memory-server-global-3.6/postinstall.js

```diff
@@ -1,22 +1,28 @@
-if (!process.env.MONGOMS_DOWNLOAD_DIR) {
+function isModuleExists(name) {
+  try {
+    return !!require.resolve(name);
+  } catch (e) {
+    return false;
+  }
+}
+
+if (!isModuleExists('../mongodb-memory-server-core/lib/util/resolve-config')) {
+  console.log('Could not resolve postinstall configuration');
+  return;
+}
+const resolveConfig = require('../mongodb-memory-server-core/lib/util/resolve-config').default;
+
+if (!resolveConfig('DOWNLOAD_DIR')) {
   process.env.MONGOMS_DOWNLOAD_DIR = require('path').resolve(
     require('os').homedir(),
     '.cache',
     'mongodb-binaries'
   );
 }
-if (!process.env.MONGOMS_VERSION) {
+if (!resolveConfig('VERSION')) {
   process.env.MONGOMS_VERSION = '3.6.12'; // don't use `latest` it's nightly build
 }
```

Comment:
I think that better will be to create some method in `resolve-config` which setup default variables.

So 
```js
const resolveConfig = require('../mongodb-memory-server-core/lib/util/resolve-config').default;

if (!resolveConfig('VERSION')) {
  process.env.MONGOMS_VERSION = '3.6.12'; // don't use `latest` it's nightly build	
}	}
```

can be rewritten on something more clear:
```js
const setDefaultValue = require('../mongodb-memory-server-core/lib/util/resolve-config').setDefaultValue;

setDefaultValue('VERSION', '3.6.12'); // don't use `latest` it's nightly build	
```

`setDefaultValue` just for example, maybe you can suggest a better name.